--- ..\..\..\..\SwProject\Source\GenDataOS\osobjs.inc.bak	2016-04-26 15:50:39.882596900 -0400
+++ ..\..\..\..\SwProject\Source\GenDataOS\osobjs.inc	2016-04-26 16:00:24.809596900 -0400
@@ -6,14 +6,26 @@
 /* Version of general code: 5.23c */
 
      . = align(16384);   /* align to region size 16384 */
-     *(.ossDataAp_10:noinit) {START(_ossAppluDataStartAp_10), SIZE(_ossAppluDataSizeAp_10)}
+     _ossAppluDataStartAp_10=.;
+     /* Nexteer patch for TypeH Data */
+     *(.ossDataAp_10:noinit:typeH) {START(Lnk_TypeH10_Start), SIZE(Lnk_TypeH10_Size)}
+     *(.ossDataAp_10:noinit)
      _ossAppluDataEndAp_10=.;
+     _ossAppluDataSizeAp_10 = _ossAppluDataEndAp_10 - _ossAppluDataStartAp_10 + 1;
      . = . + 1;
      . = align(16384); /* Use 8 subregions with size 2048*/
 
      . = align(16384);   /* align to region size 16384 */
-     *(.ossDataAp_9:noinit) {START(_ossAppluDataStartAp_9), SIZE(_ossAppluDataSizeAp_9)}
+     _ossAppluDataStartAp_9=.;
+     /* Nexteer patch for TypeH Data */
+     *(.ossDataAp_9:noinit:typeH) {START(Lnk_TypeH9_Start), SIZE(Lnk_TypeH9_Size)}
+     *(.ossDataAp_9:noinit)
+     _Hidden_Ram_Start = .;
+     *(.hidden)
+     _Hidden_Ram_End = .;
+     *(.bss)
      _ossAppluDataEndAp_9=.;
+     _ossAppluDataSizeAp_9 = _ossAppluDataEndAp_9 - _ossAppluDataStartAp_9 + 1;
      . = . + 1;
      . = align(16384); /* Use 8 subregions with size 2048*/
 
@@ -204,8 +216,12 @@
      . = align(1024); /* Use 8 subregions with size 128*/
 
      . = align(512);   /* align to region size 512 */
-     *(.ossDataAp_8:noinit) {START(_ossAppluDataStartAp_8), SIZE(_ossAppluDataSizeAp_8)}
+     _ossAppluDataStartAp_8=.;
+     /* Nexteer patch for TypeH Data */
+     *(.ossDataAp_8:noinit:typeH) {START(Lnk_TypeH8_Start), SIZE(Lnk_TypeH8_Size)}
+     *(.ossDataAp_8:noinit)
      _ossAppluDataEndAp_8=.;
+     _ossAppluDataSizeAp_8 = _ossAppluDataEndAp_8 - _ossAppluDataStartAp_8 + 1;
      . = . + 1;
      . = align(512); /* Use 8 subregions with size 64*/
 
@@ -240,8 +256,12 @@
      . = align(256); /* Use 8 subregions with size 32*/
 
      . = align(0);   /* align to region size 0 */
-     *(.ossDataAp_0:noinit) {START(_ossAppluDataStartAp_0), SIZE(_ossAppluDataSizeAp_0)}
+     _ossAppluDataStartAp_0=.;
+     /* Nexteer patch for TypeH Data */
+     *(.ossDataAp_0:noinit:typeH) {START(Lnk_TypeH0_Start), SIZE(Lnk_TypeH0_Size)}
+     *(.ossDataAp_0:noinit)
      _ossAppluDataEndAp_0=.;
+     _ossAppluDataSizeAp_0 = _ossAppluDataEndAp_0 - _ossAppluDataStartAp_0 + 1;
      . = . + 1;
      . = align(0); /* Use no subregions */
 
